bedrock/teach/templates/teach/smarton/security.html

```diff
@@ -0,0 +1,327 @@
+          </ul>
+          <ul>
+            <li><a href="https://support.apple.com/HT204416" class="go">{{ _('Safari update') }}</a></li>
+            <li><a href="https://support.apple.com/HT202819" class="go">{{ _('Safari plugin check') }}</a></li>
+          </ul>
+        </div>
+      </div>
+
+      <br style="clear:both;">
+
+      <img class="block-hero" width="220" height="220" src="{{ static('img/sandstone/grain.png') }}" alt="">
+      <h3>{{ _('Install a security system') }}</h3>
+      <p>
+        {{ _('Now that you’ve handled the basics, set the alarm.') }}
+        {{ _('These programs prevent and protect against cyber surprises.') }}
+      </p>
+
+      <div class="columns columns-2 with-graphic">
+        <div class="col">
+          <h4>{{ _('Anti-virus software') }}</h4>
+          <p>
+            {{ _('It won’t protect against everything, but it’s a good start.') }}
+            {{ _('Here are two free options:') }}
+          </p>
+          <ul>
+            <li><a href="https://www.avast.com/">Avast</a></li>
+            <li><a href="http://www.bitdefender.com/solutions/free.html">BitDefender</a></li>
+          </ul>
+        </div>
+
+        <div class="col">
+          <h4>{{ _('Firefox Add-ons') }}</h4>
+          <p>{{ _('These add-ons will help you identify what you’re clicking before you click:') }}</p>
+          <ul>
+            <li>
+              {{ _('<a href="{url}">Long URL please</a> — replaces most shortened URLs with the originals so you know the destination').format(url='https://addons.mozilla.org/firefox/addon/long-url-please/')|safe }}
+            </li>
+            <li>
+              {{ _('<a href="{url}">URL Tooltip</a> — displays the destination of a link when hovering over it').format(url='https://addons.mozilla.org/firefox/addon/url-tooltip/')|safe }}
+            </li>
+          </ul>
+        </div>
+      </div>
+
+    </div>
+  </section>
+
+  <section id="chat" class="section section-chat">
+    <div class="content">
+
+      <h2 class="section-title">{{ _('Blast your cybersecurity smarts') }}</h2>
+      <p class="section-intro">
+        {{ _('The more we talk, share tips and stay alert about cybersecurity, the safer we all are.') }}
+        {{ _('Keep the conversation going with your community on social media, at home — maybe even on that public Wi-Fi at the airport.') }}
+      </p>
+
+      <aside class="callout">
+        <p>{{ _('Connect with your community to explore topics related to privacy and tracking by joining or starting a <a href="{url}">Mozilla Club</a>.').format(url='https://teach.mozilla.org/clubs/')|safe }}</p>
+      </aside>
+
+      <aside class="callout">
+        <p>{{ _('Are you a teacher? Give your students a <a href="{url}">simple lesson on government surveillance</a>.').format(url='https://teach.mozilla.org/activities/')|safe }}</p>
```

Comment:
Checked the copy deck, it should be "Are you a teacher? Give your students a simple lesson in cybersecurity."